client/packages/lowcoder/src/i18n/locales/en.ts

```diff
@@ -2092,12 +2092,12 @@ export const en = {
     aboutUs: "",
     changeLog: "",
     introVideo: "",
-    devNpmPlugin: "https://docs.lowcoder.dev/build-plugins/develop-your-first-plugin",
+    devNpmPlugin: "https://docs.lowcoder.cloud/build-plugins/develop-your-first-plugin",
     devNpmPluginText: "How to develop npm plugin",
-    useHost: "https://docs.lowcoder.dev/self-hosting/access-local-database-or-api",
+    useHost: "https://docs.lowcoder.cloud/self-hosting/access-local-database-or-api",
```

Comment:
https://docs.lowcoder.cloud/setup-and-run/self-hosting/access-local-database-or-api